//
// Copyright 2020 Google LLC
//
// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at
//
//      http://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS,
// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
// See the License for the specific language governing permissions and
// limitations under the License.
//

syntax = "proto2";

package parameter_grammar;

message Whitespace {
    enum Type {
        SPACE = 0;
        BACKSPACE = 1;
        TAB = 2;
        NEWLINE = 3;
    }
    required Type space = 1;
    repeated Type additional = 2;
}

message Default {
    required bytes content = 1;
}